app/assets/javascripts/accounts.js

```diff
@@ -110,6 +110,8 @@
     $('#accounts_list_search').focus();
     if (Util.isIOSApp()) {
       MobileApp.savePasswordForTouchID();
+    } else if (Util.isAndroidApp()) {
+      AndroidJs.savePasswordForTouchID(userData.userId, userData.masterPassword, userData.touchIdEnabled);
```

Comment:
@scottsatkin These are calls to new methods implemented in the Android app changes. Do you know what would happen if we deploy the server-side changes before releasing the new Android app? I want to make sure that the existing Android app wont crash/break after these changes are deployed. We might have to check the app version as well.

Reply:
Nice catch -- old versions of the app would fails to load after the server is updated!  I just confirmed that.

I've updated the pull request in f757a7e to wrap calls to new AndroidJs functions to prevent old app versions from crashing.